### PR TITLE
ISPN-5538 Add test for JGroups properties-based CallbackHandler

### PR DESCRIPTION
**Pull request by vjuranek**
https://issues.jboss.org/browse/ISPN-5538

### CODE REVIEW

File: integrationtests/security-it/src/test/resources/arquillian.xml [hunk 1]
```diff
@@ -74,6 +74,43 @@
...
+                                             -Dsasl.credentials.properties=${basedir}/target/test-classes/jgroups-simple-auth.properties
+                                             -Dsasl.local.principal=test_user
+                                             -Dsasl.realm=test_realm
+            </property>
+            <property name="managementPort">10090</property>
+         </configuration>
+      </container>
+      
+      <container qualifier="simple-auth-node1" mode="manual">
+         <protocol type="jmx-as7">
```
Comment (danberindei) [inline integrationtests/security-it/src/test/resources/arquillian.xml]: Should use the `Servlet 3.0` protocol I think: https://issues.jboss.org/browse/ISPN-5597

**Comment by vjuranek** [comment]
Ops, I wasn't aware about this issue, thanks Dan! It's fixed now.

File: integrationtests/security-it/src/test/java/org/infinispan/test/integration/security/embedded/NodeAuthPropertiesHandler.java [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+/**
+ * 
+ * @author vjuranek
+ * @since 8.0
+ */
+@RunWith(Arquillian.class)
+public class NodeAuthPropertiesHandler extends AbstractNodeAuthentication {
```
Comment (danberindei) [inline integrationtests/security-it/src/test/java/org/infinispan/test/integration/security/embedded/NodeAuthPropertiesHandler.java]: Could you rename the test to have an IT at the end? Please also add a blurb in the javadoc.

Comment (vjuranek) [inline integrationtests/security-it/src/test/java/org/infinispan/test/integration/security/embedded/NodeAuthPropertiesHandler.java]: Fixed now

**Comment by danberindei** [comment]
Thanks Vojtech, integrated! I saw some test failures in the last build, but they don't seem related.

**Pull request closed without merging**